docs/cases/api/cases-api/cases-api-add-comment.asciidoc

```diff
@@ -53,57 +64,65 @@ POST api/cases/293f1bc0-74f6-11ea-b83a-553aecdb28b6/comments
 
 ==== Response payload
 
-The updated JSON case object with the new comment, the user who created the
-comment, and the comment's ID, version, and creation time.
+The response contains an updated JSON case object with the newly added comment, the user who created it, and the comment's ID, version, and creation time.
 
 ===== Example response
 
 [source,json]
 --------------------------------------------------
 
-  "id": "293f1bc0-74f6-11ea-b83a-553aecdb28b6",
-  "version": "WzIwNDMyLDFd",
-  "comments": [
+{
+  "comments":[
     {
       "id": "8af6ac20-74f6-11ea-b83a-553aecdb28b6",
       "version": "WzIwNDMxLDFd",
-      "type": "user",
-      "comment": "That is nothing - Ethan Hunt answered a targeted social media campaign promoting phishy pension schemes to IMF operatives.",
-      "created_at": "2020-04-02T15:28:03.034Z",
+      "type":"user",
+      "owner":"cases",
+      "comment":"That is nothing - Ethan Hunt answered a targeted social media campaign promoting phishy pension schemes to IMF operatives.",
+      "created_at":"2022-03-24T00:49:47.716Z",
```

Comment:
These pop culture references are a little too cute for me, but there's like half a dozen others throughout the page, so revising them is probably out of scope for the fix you're doing here. 

(The references also jump franchises, from Mission Impossible to James Bond, which is problematic for other, nerdier reasons. 🤪 )

Reply:
See, I always thought this was the start of a crossover series where Ethan Hunt went rogue because he was tired of being given impossible missions. ;) 

Jokes aside, I might do a sweeping update to these examples at some point to refresh 'em but I don't mind them for now. :)